CHANGELOG.md

```diff
@@ -1,14 +1,28 @@
-# dbt_amplitude_source version.version
+# dbt_amplitude_source v0.4.0
+This release includes the following updates:
+
+## Breaking Change
+- Filtered events to include only those with `event_time` up to and including the current date, preventing data quality issues in downstream incremental models. Future events are treated as erroneous. ([#14](https://github.com/fivetran/dbt_amplitude_source/pull/14))
```

Comment:
Can we include a mention of the model where this applied filter has been modified.

Reply:
Yes--updated!